apps/dashboard/src/components/workflow-editor/steps/digest/digest-key.tsx

```diff
@@ -22,18 +30,21 @@ export const DigestKey = () => {
                 <span>Aggregated by</span>
               </span>
             </FormLabel>
-            <InputFieldPure className="h-7 rounded-lg border pr-0">
+            <InputFieldPure className="h-7 items-center gap-0 rounded-lg border">
               <FormLabel className="flex h-full items-center gap-1 border-r border-neutral-100 pr-1">
                 <Code2 className="-ml-1.5 size-5" />
-                <span className="text-foreground-600 mt-[1px] text-xs font-normal">subscriberId</span>
+                <span className="text-foreground-600 text-xs font-normal">subscriberId</span>
               </FormLabel>
               <FormControl>
-                <Input
-                  className={cn(
-                    'min-w-[20ch] [appearance:textfield] [&::-webkit-inner-spin-button]:appearance-none [&::-webkit-outer-spin-button]:appearance-none'
-                  )}
+                <Editor
```

Comment:
Note that as part of future work, we need to introduce a single line editor component that leverages the `<Editor/>` but behaves like an input. That is, on Enter press no new limes should be added, and on tab press we should move to the next input in the form instead of applying indentation to the current line of text.